src/main/resources/completion.txt

```diff
@@ -0,0 +1,17 @@
+between
```

Comment:
What's the source you referred to build this list? (I think it's not a full list of SQL keywords?)

Reply:
I just picked some reserved words which have length >= 5 and frequently used (criteria: just my opinion) from https://docs.actian.com/psql/PSQLv13/index.html#page/sqlref/sqlkword.htm (top page I googled "sql reserved words")

Reply:
> I just picked some reserved words which have length >= 5

I see, it sounds reasonable from the perspective of how useful it is to complete, but maybe it makes some people to feel inconsistent?

E.g, if somewhere writes `i` and type TAB, then the candidate doesn't show `in` which is totally valid sql keyword but feels like lacking in candidate (= valid words to write)?

Reply:
Hm, that's right.

Then I think we should omit sql keywords suggestion for now? I think just tables and columns are even helpful and sql keywords could be too noisy

Reply:
ok, that'd be an option too.